sys/include/net/cord/epsim.h

```diff
@@ -51,6 +51,10 @@ int cord_epsim_register(void);
  * @brief   Spawn a new thread that registers the node and updates the
  *          registration with all responding RDs using the simple registration
  *          process
+ *
+ * @note    Only available with @ref cord_epsim_standalone compiled in
+ * @note    This function must only be called once (typically during system
```

Comment:
I think this should rather be a `@warning` or `@attention` even.

Reply:
@warning seems sensible. Will also adapt that in `cord/ep_standalone.h`.